### ISSUE TITLE
Windows: Slow start and white window

### ISSUE THREAD

**Issue by ogoffart** [slint-ui/slint#2254]
Following a discussion on mattermost, it was reported that on window, the window first appear white, and then it takes a long time until the first frame appear

**Comment by ogoffart** [comment]
I can reproduce the problem,  this is with the winit backend. The Qt backend works as expected.

The window is shown before the event loop is started. It appear that `GLWindow::show` takes some time to run, then the event loop starts.
We were told that other framework using winit workaround the glitch by keeping the window hidden until the init phase is done <https://github.com/iced-rs/iced/pull/1698>

**Comment by tronical** [comment]
I can sort of reproduce this. I've made clips of this with the different renderers. This is a release build:

Qt:

https://user-images.githubusercontent.com/1486/227491782-fb8cc27b-4315-4a9f-bda6-f1f85f67f61d.mov

Softbuffer:

https://user-images.githubusercontent.com/1486/227491836-6dc0008c-1b84-4bcb-80ab-16f92114b2e3.mov

Skia (D3D):

https://user-images.githubusercontent.com/1486/227491861-876ef502-f684-49bd-990d-06a0e3252fd2.mov

Skia (Glutin):


https://user-images.githubusercontent.com/1486/227492675-014ab40c-0e3d-4797-aab7-96c82cbcf978.mov


Glutin/FemtoVG:

https://user-images.githubusercontent.com/1486/227491921-fc1270ef-a716-4867-a224-bf2bf1821719.mov

**Comment by tronical** [comment]
The above clips suggest to me that this is more related to the way OpenGL rendering happens on Windows (and for example the driver situation), that it is about winit.

**Comment by tronical** [comment]
It might be possible to change the startup sequence in the winit backend from

```rust
let window_builder = ...
let window = window_builder.build();
...

fn paint_event()
{
    render_into_window(window);
}
```

to

```rust
let window_builder = ...
let window_builder = window_builder.with_visible(false);
let window = window_builder.build();
render_into_window(window); // calls swapBuffers
window.set_visible(true);
...

fn paint_event()
{
    render_into_window(window);
}
```

**Comment by ogoffart** [comment]
@tronical could that be fixed by https://github.com/slint-ui/slint/pull/12143 ?
